kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/sink/config/PartitionField.scala

```diff
@@ -97,6 +99,8 @@ case class TopicPartitionField() extends PartitionField {
 
 case class PartitionPartitionField() extends PartitionField {
```

Comment:
The naming is great :) partitionpartition :)

Reply:
🥔 🥔 🥔